### PR TITLE
Create: Faq document

### PR DESCRIPTION
**Pull request by EnesDONER**
New feature added to Cms kit pro

### CODE REVIEW

File: en/images/cmskit-module-faq-edit-question-page.png
Comment (ebicoglu) [inline en/images/cmskit-module-faq-edit-question-page.png:1]: image

File: en/images/cmskit-module-faq-edit-page.png
Comment (ebicoglu) [inline en/images/cmskit-module-faq-edit-page.png:1]: why Order field is not numeric on Edit FAQ? In the Edit Question dialog it looks numeric field.
image
image

File: en/modules/cms-kit/faq.md
Comment (ebicoglu) [inline en/modules/cms-kit/faq.md:1]: Always use **FAQ** not **faq** in the documents.
image

Comment (ebicoglu) [reply en/modules/cms-kit/faq.md:1]: image

**Review by ebicoglu — CHANGES_REQUESTED** [review]

File: en/images/cmskit-module-faq-edit-page.png
Comment (EnesDONER) [reply en/images/cmskit-module-faq-edit-page.png:1]: I use the abp input. These buttons appear when you hover over the input. Just like:
error

**Review by ebicoglu — APPROVED** [review]

**Pull request merged**